lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/evaluate-scores.yaml

```diff
@@ -9,6 +9,11 @@ states:
       next:
         targetState: EVALUATE_GPG45_SCORES
 
+  START_NO_STORE:
```

Comment:
Do you think it makes sense to have this in here or `failed.yaml`? I'm not totally sure, but I might slightly lean towards failed, since it's not 'completing' a journey and the states are totally distinct (apart from 'return to RP', which doesn't really count).

Perhaps what I really want is for this to be `new-identity-success.yaml` or something similar 🤔 

For reference, the existing reuse journey does not use this subjourney!

Reply:
I started adding the states in `failed.yaml` but if felt weird - because they weren't about failing so I shifted them here.

`new-identity-success.yaml` - I think that would confuse things. These new states are for an old identity?

Reply:
Yeah sorry I meant, if this file were called `new-identity-success.yaml` then its purpose would be clear, and the answer would be obvious (that the new states shouldn't be in it!)

I suppose the alternative is something like `return-identity` or something, but then it's odd that normal reuse journeys don't use it.

(at the moment 'evaluate-scores' is a bit unclear to me, as it's not just about evaluating scores, but storing the result and returning to the RP)